coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetViewHelperServiceImpl.java

```diff
@@ -205,4 +207,15 @@ public void setKcBusinessRulesEngine(KcBusinessRulesEngine kcBusinessRulesEngine
 		this.kcBusinessRulesEngine = kcBusinessRulesEngine;
 	}
     
+	public boolean isBudgetMarkedForSubmission() {
+		ProposalBudgetForm form = (ProposalBudgetForm)ViewLifecycle.getModel();
+		boolean budgetMarkedForSubmission = false;
+		Budget finalBudget = form.getDevelopmentProposal().getFinalBudget();
```

Comment:
Seems like this could go in a service somewhere.  Then have the view helper service call that service.